go.mod

```diff
@@ -23,6 +23,7 @@ require (
 	github.com/keepeye/logrus-filename v0.0.0-20190711075016-ce01a4391dd1
 	github.com/klauspost/compress v1.16.4
 	github.com/lestrrat-go/file-rotatelogs v2.4.0+incompatible
+	github.com/lf-edge/ekuiper/extensions v0.0.0-20231016084259-4bf285c7c5ff
```

Comment:
This may need to replace with local version because we have make them mono repo.